assignment.js

```diff
@@ -66,6 +72,15 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const numbers = [3, 48, 29, 12, 18]
+let currentLargestNumber = 0
+for (let n = 0; n <= numbers.length; n++) {
+if (numbers[n] > currentLargestNumber) {
```

Comment:
@annayensietopping be careful with indentation here...the `if` statement should be indented one level to the right since it's inside of the `for` loop